configurations/tracking_only.yml

```diff
@@ -1,3 +1,3 @@
 features:
-- beampipe
-- tracking
+  beampipe:
```

Comment:
If `_only` is to mean 'no beampipe' then we should be consistent.
```suggestion
```

Reply:
Do you want it fixed with this pr or a separate one?